### ISSUE TITLE
webApi post and delete stopped working for non-admin

### ISSUE THREAD

**Issue by eXistenZe** [2sic/2sxc#1441]
**I'm submitting a ...**
[x] bug report

**...about**
[x] APIs like REST

**Current behavior**
webApi.post and webApi.delete started giving permission errors in one of the latest updates.
When logged as admin they work, but for normal users, they do not.
This happens in apps that have been working for several months now.
I double checked the permissions (even tried anonymous -> CRUD).
2sxc.api.min.js is included.

**Expected behavior**
$2sxc(@Dnn.Module.ModuleID).webApi.post('app/auto/content/Entity', {}, newItem);
$2sxc(@Dnn.Module.ModuleID).webApi.delete('app/auto/content/Entity', id);

These functions have always worked for my users, but recently (not sure when, but I would say in the last 2 weeks) they give the following errors:

```
Request URL:https://site.pt/API/2sxc/app/auto/content/Entity?
Request Method:POST
Status Code:401 Error in 2sxc Content API - not allowed
Remote Address:x.x.168.180:443
Referrer Policy:no-referrer-when-downgrade
Response Headers
view source
Cache-Control:no-cache
Content-Length:124
Content-Type:text/plain; charset=utf-8
Date:Wed, 20 Dec 2017 16:23:02 GMT
Expires:-1
Pragma:no-cache
Set-Cookie:dnn_IsMobile=False; path=/; HttpOnly
X-Frame-Options:SAMEORIGIN
X-Powered-By-Plesk:PleskWin
X-XSS-Protection:1; mode=block
Request Headers
view source
Accept:application/json, text/javascript, */*; q=0.01
Accept-Encoding:gzip, deflate, br
Accept-Language:en-US,en;q=0.9,pt;q=0.8,es;q=0.7,pt-PT;q=0.6
Connection:keep-alive
Content-Length:500
Content-Type:application/json
ContentBlockId:1084
Cookie:dnn_IsMobile=False; .ASPXANONYMOUS=I5-RO_VX-_WxAM-Kbg6FvvK0wC-Q7MDb7hi0lCPlcQmxB0u_D-a3JMX-XBKqULVVCWqaIN9UbJqbGX9purC7Lz2RM_tU6JGnIwd2pmA0QaHv8rZo0; __RequestVerificationToken=6Tj6LlRz7T7yjQNFVlWU-bPDzJn3Ih1ty8Q9PCnCcnW-vU_Aa9pcG1ViLHvW_NIwuBK6GQ2; USERNAME_CHANGED=; LogGUID=37a0d22c-5df4-4828-9483-0a70536ac04b; language=pt-PT; authentication=DNN; .DOTNETNUKE=1B7614FBB6F730BBBE9086228DF8E00E6F274D071C6B709675C1AC7BE774D702F903D37EF9BF0669170B6971E855F8C5273BEDC725CB9A12204CC75D386463973CC63B24B794FED7034B5794; SiteGroup=; LastPageId=1:283
DNT:1
Host:site.pt
ModuleId:1084
Origin:https://site.pt
Referer:https://site.pt/Page/Page/Page
RequestVerificationToken:LM-xWxMeDP6d-Zq3L0Loab24_mEvm_Sr-AstXziSJICayIVeDcjkJ4lBAPfXuZSeUkhohsH6UbXOODK40
TabId:280
User-Agent:Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.84 Safari/537.36
X-Requested-With:XMLHttpRequest
Request Payload
view source
{user: "enfJoao", udd: "x", nome: "", hb: "", data: "2017-12-20", notas: "", a1: "", b1: "", b2: "",…}


Request URL:https://site.pt/API/2sxc/app/auto/content/Entity
Request Method:DELETE
Status Code:405 Method Not Allowed
Remote Address:94.46.168.180:443
Referrer Policy:no-referrer-when-downgrade
Response Headers
view source
Allow:GET,POST
Cache-Control:no-cache
Content-Length:75
Content-Type:application/json; charset=utf-8
Date:Wed, 20 Dec 2017 16:24:51 GMT
Expires:-1
Pragma:no-cache
Set-Cookie:dnn_IsMobile=False; path=/; HttpOnly
X-Frame-Options:SAMEORIGIN
X-Powered-By-Plesk:PleskWin
X-XSS-Protection:1; mode=block
Request Headers
view source
Accept:application/json, text/javascript, */*; q=0.01
Accept-Encoding:gzip, deflate, br
Accept-Language:en-US,en;q=0.9,pt;q=0.8,es;q=0.7,pt-PT;q=0.6
Connection:keep-alive
Content-Type:application/json
ContentBlockId:1096
Cookie:dnn_IsMobile=False; .ASPXANONYMOUS=I5-RO_VX-_WxAM-Kbg6FvvK0wC-Q7MDb7hi0lCPlcQmxB0u_D-a3JMX-XBKqULVVCWqaIN9UbJqbGX9purC7Lz2RM_tU6JGnIwd2pmA0QaHv8rZo0; __RequestVerificationToken=6Tj6LlRz7T7yjQNFVlWU-bPDzJn3Ih1ty8Q9PCnCcnW-vU_Aa9pcG1ViLHvW_NIwuBK6GQ2; USERNAME_CHANGED=; LogGUID=37a0d22c-5df4-4828-9483-0a70536ac04b; language=pt-PT; authentication=DNN; .DOTNETNUKE=1B7614FBB6F730BBBE9086228DF8E00E6F274D071C6B709675C1AC7BE774D702F903D37EF9BF0669170B6971E855F8C5273BEDC725CB9A12204CC75D386463973CC63B24B794FED7034B5794; SiteGroup=; LastPageId=1:283
DNT:1
Host:investigar.pt
ModuleId:1096
Origin:https://site.pt
Referer:https://site.pt/Page/Page/Page
RequestVerificationToken:p2ri76qYrwYlOORagSUxrmSp8ksOdMfko8LzTAWHnhdXVVSz-cZL7F20FBAkIS0n0-T87OGJQb_HlSZ80
TabId:283
User-Agent:Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.84 Safari/537.36
X-Requested-With:XMLHttpRequest
```

**Minimal reproduction of the problem with instructions**
Use the webApi like described above.

* **2sxc version(s):** 9.13
* **Browser:** [all]
* **DNN:** [9.1.1 ] 
* **Language:** [English]

**Comment by eXistenZe** [comment]
Fixed in 9.14 :)

**Comment by iJungleboy** [comment]
...exactly ;)
